open-banking-accelerator/components/event-notifications/com.wso2.openbanking.accelerator.event.notifications.service/src/test/resources/testng.xml

```diff
@@ -1,5 +1,5 @@
 <!--
- ~ Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
This should be 2023-2024

Reply:
Fixed with https://github.com/wso2/financial-open-banking/pull/97/commits/1314cee594e93059b465e551698a11b7fa677ffe